<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menuAdd</class>
 <widget class="QDialog" name="menuAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#menuAdd{
background-color: rgb(8, 36, 8);
border-radius:5px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item row="5" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>53</horstretch>
       <verstretch>25</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-color: rgb(8, 36, 8);
border-radius:15px;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLineEdit" name="nameEdit">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(64, 64, 64);
    color: rgb(144, 238, 144);
    border-radius: 5px;
 padding: 3px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="idEdit">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(64, 64, 64);
    color: rgb(144, 238, 144);
    border-radius: 5px;
 padding: 3px;</string>
        </property>
        <property name="inputMask">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="priseEdit">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(64, 64, 64);
    color: rgb(144, 238, 144);
    border-radius: 5px;
 padding: 3px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>70</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <widget class="QWidget" name="widget_3" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(8, 36, 8);
border-radius:5px;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="styleSheet">
         <string notr="true">QPushButton {
        background-color: #2f4f2f;
       
	color: rgb(192, 191, 188);
        border: 1px solid #4caf50;
        border-radius: 8px;
        padding: 5px 10px;
    }

QPushButton:hover {
        background-color: #3e8e41;
    }</string>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="styleSheet">
         <string notr="true">QPushButton {
        background-color: #2f4f2f;
       
	color: rgb(192, 191, 188);
        border: 1px solid #4caf50;
        border-radius: 8px;
        padding: 5px 10px;
    }

QPushButton:hover {
        background-color: #3e8e41;
    }</string>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(8, 36, 8);
border-radius:5px;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>7</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 600 13pt &quot;URW Gothic&quot;;
background-color: rgba(106, 82, 177, 0);
border-right:2px  solid rgb(61, 56, 70);
border-radius:0px;
color: rgb(144, 238, 144);</string>
        </property>
        <property name="text">
         <string>Select Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 600 13pt &quot;URW Gothic&quot;;
background-color: rgba(106, 82, 177, 0);
border-right:2px  solid rgb(61, 56, 70);
border-radius:0px;
color: rgb(144, 238, 144);</string>
        </property>
        <property name="text">
         <string>Select Id</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: 600 13pt &quot;URW Gothic&quot;;
background-color: rgba(106, 82, 177, 0);
border-radius:0px;
color: rgb(144, 238, 144);</string>
        </property>
        <property name="text">
         <string>Select Price</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_4" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(8, 36, 8);
border-radius:5px;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>11</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 100 italic 22pt &quot;Ubuntu Sans&quot;;
background-color: rgb(8, 36, 8);
border-radius:5px;</string>
        </property>
        <property name="text">
         <string>Add Product</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
